packages/@uppy/aws-s3/src/MiniXHRUpload.js

```diff
@@ -237,7 +237,7 @@ module.exports = class MiniXHRUpload {
       xhr.open(opts.method.toUpperCase(), opts.endpoint, true)
       // IE10 does not allow setting `withCredentials` and `responseType`
       // before `open()` is called.
```

Comment:
```suggestion
// before `open()` is called.
// it’s important to set withCredentials to true or false, not undefinied, otherwise React Native crashes
```